.github/BOTMETA.yml

```diff
@@ -855,7 +855,7 @@ files:
       - inventory
     labels:
       - cloud
-  lib/ansible/plugins/inventory/netbox.py: sieben
+  lib/ansible/plugins/inventory/netbox.py: sieben Anthony25
```

Comment:
There is no need to add module authors in botmeta (by default, module authors are maintainers), remove the whole entry.